status-im/status-react
# resources: added more games to default contacts

fixes #868 Added more games to default contacts

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
Implemented feature request to add more games to the default list of contacts. New entries created in *resources/default_contacts.json* along with logo images

### Steps to test:
- Open Status
- Add Contacts
- See new entries for Vdice, Roulette, Blockjack and Etheroll

status: ready
